db.sql

```diff
@@ -1,6 +1,6 @@
 -- MySQL dump 10.19  Distrib 10.3.28-MariaDB, for Linux (x86_64)
 --
--- Host: localhost    Database: sev_attest
+-- Host: gensev2.sl.cloud9.ibm.com    Database: sev_attest
```

Comment:
I don't think we want this in the repo.

Reply:
I removed this.  You will see it in the next push.  This was inserted when we dumped the database after removing the secpol table.